opencog/combo/converter/combo_atomese.cc

```diff
@@ -117,8 +117,9 @@ std::pair<Type, Handle> ComboToAtomeseConverter::atomese_vertex(const vertex &v,
 	return std::make_pair(type, handle);
 }
 
-Handle ComboToAtomeseConverter::operator()(const combo_tree &ct)
+Handle ComboToAtomeseConverter::operator()(const combo_tree &ct, std::string suffix)
```

Comment:
It's better to pass suffix by const reference. Maybe gcc can optimize it but just as a habit.